snuba/web/views.py

```diff
@@ -389,7 +389,10 @@ def write(*, dataset: Dataset):
                 .get_stream_loader()
                 .get_processor()
                 .process_message(
-                    message, KafkaMessageMetadata(offset=offset, partition=0,)
+                    message,
+                    KafkaMessageMetadata(
+                        offset=offset, partition=0, timestamp=datetime.utcnow()
```

Comment:
This is `utcnow` to mirror everything else, especially: https://github.com/getsentry/snuba/blob/ad88f1c55cba18523a4bf9bd514fc1b26fcf6ebb/snuba/utils/streams/kafka.py#L415